extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -0,0 +1,111 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { JupyterBookSection, IJupyterBookSectionV1, IJupyterBookSectionV2 } from '../contracts/content';
+import * as path from 'path';
+
+export enum BookVersion {
+	v1 = 'v1',
+	v2 = 'v2'
+}
+
+export function getContentPath(version: string, bookPath: string, filePath: string): string {
+	return BookVersion.v1 === version ? path.posix.join(bookPath, 'content', filePath) : path.posix.join(bookPath, filePath);
+}
+
+export function getTocPath(version: string, bookPath: string): string {
+	return BookVersion.v1 === version ? path.posix.join(bookPath, '_data', 'toc.yml') : path.posix.join(bookPath, '_toc.yml');
+}
+
+export class BookVersionHandler {
```

Comment:
If these are all just going to be static members then you might as well just make them module-level functions. The class is pretty unnecessary here